app/app.js

```diff
@@ -12,8 +12,21 @@ angular.module('app', ['angular-md5'])
     var seriesVolumes = $window.seriesVolumes;
     
 
-    const INITIAL_ZERO = 0;  // Used for all zero intialised vars
+    const INITIAL_ZERO = 0;  // Used for all vars initialised to 0
+    const INITAL_ONE = 1;    // Used for counters intiialised to 1
     const BODY_PADDING = 20;
+    const LEFT_OFFSET = 200;
+    const TOP_OFFSET = 300;
+
+    //Colour constants used in multiple functions
+    const WHEEL_SIZE = 360;     //360 degrees in colour wheel
+    const STEP_CHANGE = 30;     //define how far to step around the colour wheel each time
+    const MIN_LIGHTNESS = 30;   //Lightness can be in the range 0-100
```

Comment:
What do you think about reversing the words in ones like this? It can help with grouping, e.g. `LIGHTNESS_MIN` and `LIGHTNESS_MAX` will stay together when ordered alphabetically

Reply:
Good idea.  Renamed and ordered.